### PR TITLE
Add preloading preconfigs of modules to memory

### PR DESCRIPTION
**Pull request by AhmetNSimsek**
EDT: Profiling results below.

### CODE REVIEW

**Comment by AhmetNSimsek** [comment]
```python
import time
import siibra
siibra.core.load_preconfigurations()
siibra.volumes.load_preconfigurations()
start = time.time()
siibra.get_map(parcellation="julich 2.9", space="MNI152")
print(time.time() - start)
```
| | WO preload | W map preload | W map and core preload |
| --- | --- | --- | --- |
| WO local cache | 44.7s  | 0.58s | 0.029s |
| W local cache | 5.6s | 0.27s | 0.027s |

**Review by xgui3783 — APPROVED** [review]
Looks good. 

I dont' know about the function name though.

`load_preconfigurations` -> I know in the background we quite literally loads the preconfiguration into memory. But I think it still can be called something generic, like `warmup`. This is so that in the future, we can add warm up strategies that does not involve preconfiguration

File: siibra/__init__.py [hunk 1]
```diff
@@ -135,4 +152,5 @@ def __dir__():
         "vocabularies",
         "__version__",
         "cache",
+        "preload_configurations"
```
Comment (xgui3783) [inline siibra/__init__.py:155]: Wrong export

**Review by xgui3783 — APPROVED** [review]
Just one typo, otherwise good to go.

File: siibra/__init__.py (hunk 1 above)
Comment (AhmetNSimsek) [reply siibra/__init__.py:155]: Thanks for catching!

**Pull request merged**